g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh

```diff
@@ -0,0 +1,130 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/envoyproxy/go-control-plane
+# Version       : v0.9.0
+# Source repo   : https://github.com/go-stack/stack.git
+# Tested on     : RHEL 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Amit Baheti (aramswar@in.ibm.com)
+#
```

Comment:
We have recently introduced additional fields in the script header, can you please refer to the latest templates and / or guidelines doc and include those? Mainly "Language", "Travis-Check" etc.